lib/utils/metadata.dart

```diff
@@ -1,5 +1,5 @@
-const String gitBranchName = '';
-const String gitCommitHash = '';
-const String gitCommitMessage = '';
-const String gitCommitDate = '';
-const String gitOriginUrl = '';
+const String gitBranchName = 'master';
+const String gitCommitHash = '2a86d3b81013133bac655f8495350359fccfab31';
+const String gitCommitMessage = 'Merge pull request #19 from alienc0der/bigint-support';
+const String gitCommitDate = '2023-07-20 18:21:10 +0100';
+const String gitOriginUrl = 'https://github.com/zenon-network/syrius.git';
```

Comment:
These probably shouldn't be committed?

Reply:
@Sol-Sanctum can you remove this file?

Reply:
Reverted changes.
I think this needs to be handled in the workflow.